ServiceDesk.DAL/Entities/Ticket.cs

```diff
@@ -1,4 +1,4 @@
-﻿using ServiceDesk.DAL.Enums;
+﻿using ServiceDesk.Domain.Enums;
```

Comment:
Why you changed the namespace but project still the same?

Reply:
Enums are common for entities and models, so I created new project "ServiceDesk.Domain" in solution and transfered enums to new project. And I had to change the namespace.